storage/providerregistry.go

```diff
@@ -69,3 +69,25 @@ func IsProviderSupported(envType string, providerType ProviderType) bool {
 	}
 	return false
 }
+
+type defaultStoragePool map[StorageKind]string
```

Comment:
Doesn't really matter, but I feel compelled to say that use of a map is overkill when there can be at most 2 entries :)